packages/portfolio-deploy/test/portfolio.test.ts

```diff
@@ -105,6 +105,11 @@ const exampleDynamicChainInfo = {
     reference: '43114',
     cctpDestinationDomain: 1,
   },
+  Ethereum: {
```

Comment:
it's sort of boring that this has to be updated in so many places. consider refactoring.